includes/widgets/image-carousel.php

```diff
@@ -497,10 +497,25 @@ protected function register_controls() {
 		);
 
 		$this->add_control(
-			'dots_color',
+			'dots_inactive_color',
```

Comment:
Please keep consistency
```suggestion
			'dots_color_inactive',
```

Reply:
Done